lib/dep_graph.js

```diff
@@ -324,6 +324,17 @@ DepGraph.prototype = {
 
       return result;
     }
+  },
+  entryNodes: function () {
+    const result = [];
```

Comment:
I think this would be simpler as:

```javascript
var self = this;
return Object.keys(this.nodes).filter(function (node) {
  return self.incomingEdges[node].length === 0;
});
```

(which is what `overallOrder` does to find the entry points)